lib/house.rb

```diff
@@ -0,0 +1,32 @@
+class House
+  attr_reader :price, :address, :rooms
+  def initialize(price, address)
+    @price = price.gsub('$', '').to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    false
```

Comment:
We need to compare the price here to the integer 50000 to see if the price is above market average